template/Makefile.tpl

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty file on purpose?

Reply:
+1
Makefile should be generated for initial bootstrap.